app/src/renderer/js/main.js

```diff
@@ -1,5 +1,6 @@
 import {ipcRenderer, remote} from 'electron';
 import EventEmitter from 'events';
+import {default as createAperture, audioDevices} from 'aperture';
```

Comment:
```js
import Aperture from 'aperture';
```

And then just use `Aperture.audioDevices`.

Reply:
@sindresorhus default export is `createAperture` and `Aperture` is not exported. should I send a PR to that package?

Reply:
Oh, that's true. Duuh. I can fix the docs there. Never mind then.